src/Shared/Network/Helpers/CommanderHelper.cs

```diff
@@ -42,6 +42,11 @@ public static void AddCommander(this Packet packet, ICommander commander)
 			{
 				packet.PutInt(0);
 			}
+
+			// [i170175 (2017-09-01)] ?
+			{
+				packet.PutInt(0);
```

Comment:
This appears to be some buffer on the `APPEARANCE_PC` object, but I haven't seen it used by the client yet. It may just be a buffer.

Reply:
This assumption of what it does should probably go into the code =)

Reply:
You mean a comment? Yes. It should.